src/routes.tsx

```diff
@@ -15,6 +18,34 @@ const GoodId = lazy(() => import('./pages/gd/GoodId'))
 const BuyGd = lazy(() => import('./pages/gd/BuyGD'))
 const NewsFeedPage = lazy(() => import('./pages/gd/News'))
 
+const RoutesWrapper = () => {
+    const posthog = usePostHog()
+    const [posthogInitialized, setPosthogInitialized] = useState(false)
+
+    useEffect(() => {
```

Comment:
The complicated effect should simply be replaced with
if(posthog)
{
await posthog.getFeatureFlagsAndPayloads()
setPosthogInitialized(true)
}

Reply:
Not sure I understand.
getFeatureFlagsAndPayloads is not a async function